### PR TITLE
fix(content): unsubscribe from viewCtrl observables after content ins…

### PR DESCRIPTION
**Pull request by cmorbitzer**
…tance destroyed

#### Short description of what this resolves:
Resolves error `Uncaught TypeError: Cannot read property 'ev' of null` when user switches views too quickly

#### Changes proposed in this pull request:
The subscribers on the viewCtrl observables can resolve after the Content instance has been destroyed. This results in the _scroll property being undefined at the time the subscriber resolves, causing this error. This may also be related to #9593 

**Ionic Version**: 2.x

**Fixes**: #10045

### CODE REVIEW

File: src/components/content/content.ts [hunk 1]
```diff
@@ -400,6 +404,8 @@ export class Content extends Ion implements OnDestroy, OnInit {
    */
   ngOnDestroy() {
     this._scLsn && this._scLsn();
+    this._viewCtrlReadSub && this._viewCtrlReadSub.unsubscribe();
+    this._viewCtrlWriteSub && this._viewCtrlWriteSub.unsubscribe();
     this._scroll && this._scroll.destroy();
```
Comment (manucorporat) [inline src/components/content/content.ts]: Can you add this._viewCtrlReadSub = this._viewCtrlWriteSub = null?
just for completeness

**Comment by manucorporat** [comment]
I also submitted this PR:
https://github.com/driftyco/ionic/pull/10060/files

Now I have noticed my PR does not fix the issue, but it might help. We are subscribing to the event before this.scroll is actually instanced. 

I think it is safer to initialize this._scroll before.

**Comment by cmorbitzer** [comment]
Both changes added

**Comment by manucorporat** [comment]
Merged! thanks a lot for the PR!

**Pull request merged**